modules/server/src/main/ts/bedrock/server/RunnerRoutes.ts

```diff
@@ -41,10 +47,32 @@ export const generate = async (mode: string, projectdir: string, basedir: string
     }
   };
 
-  const workspaceRoots = (
+  const findPnpmWorkspaces = async (): Promise<WorkspaceRoot[]> => {
+    if (!fs.existsSync(path.join(projectdir, 'pnpm-workspace.yaml'))) {
+      return [];
+    }
+
+    return new Promise<WorkspaceRoot[]>((resolve, reject) =>
+      childProcess.exec('pnpm list -r --only-projects --json', (err, stdout, stderr) => {
+        if (err) reject(err);
+        else if (stderr) reject(stderr);
```

Comment:
This doesn't necessarily mean the command failed 🤔 

`stderr` should be printed, though, regardless of error state.

Reply:
`stderr` is just error logged now. It only rejects if there's an `err`